### PR TITLE
Ben update nqa demo

### PR DESCRIPTION
**Pull request by Ben-CHATA**
- Added yes/no response to AutoAE QandA responses

### CODE REVIEW

File: src/components/QueryOutput/QueryOutput.js [hunk 1]
```diff
@@ -1876,6 +1881,17 @@ export default class QueryOutput extends React.Component {
     const { displayType } = this.state
     const { queryResponse } = this.props
 
+    if (
+      getAuthentication(this.props.authentication).isQandA &&
```
Comment (Shiv-Chata) [inline src/components/QueryOutput/QueryOutput.js:1885]: Should use _.get(getAuthentication(this.props.authentication), isQandA) to be safe

File: src/components/QueryOutput/QueryOutput.js [hunk 2]
```diff
@@ -1876,6 +1881,17 @@ export default class QueryOutput extends React.Component {
...
+      this.state.QandASuggestions.length !== 0
+    ) {
+      return this.renderSuggestionMessage(
+        this.state.QandASuggestions,
+        _get(queryResponse, 'data.data.query_id'),
+        true // isQandA
```
Comment (Shiv-Chata) [inline src/components/QueryOutput/QueryOutput.js:1891]: remove comment

File: src/components/QueryOutput/QueryOutput.js [hunk 3]
```diff
@@ -2022,6 +2089,8 @@ export default class QueryOutput extends React.Component {
           ${this.state.displayType === 'html' ? 'html-content' : ''}`}
         >
           {this.renderResponse(width, height)}
+          {getAuthentication(this.props.authentication).isQandA &&
```
Comment (Shiv-Chata) [inline src/components/QueryOutput/QueryOutput.js:2092]: use _.get as mentioned above

File: src/js/queryService.js [hunk 4]
```diff
@@ -348,11 +377,12 @@ export const sendSuggestion = ({
   apiKey,
   domain,
   token,
+  isQandA,
 }) => {
   const url = `${domain}/autoql/api/v1/query/${queryId}/suggestions?key=${apiKey}`
   const data = { suggestion }
 
-  if (!token || !domain || !apiKey) {
+  if (!isQandA && (!token || !domain || !apiKey)) {
```
Comment (Shiv-Chata) [inline src/js/queryService.js:385]: Maybe out (!token || !domain || !apiKey) into a var and then use in the var so the If statement isnt to large. Just a suggestion though.

File: src/components/QueryOutput/QueryOutput.js (hunk 1 above)
Comment (Ben-CHATA) [reply src/components/QueryOutput/QueryOutput.js:1885]: done

File: src/components/QueryOutput/QueryOutput.js (hunk 2 above)
Comment (Ben-CHATA) [reply src/components/QueryOutput/QueryOutput.js:1891]: done

File: src/js/queryService.js (hunk 4 above)
Comment (Ben-CHATA) [reply src/js/queryService.js:385]: Interesting suggestion, for this case I don't really feel like it's necessary as all of the other functions follow this format.

File: src/components/QueryOutput/QueryOutput.js (hunk 3 above)
Comment (Ben-CHATA) [reply src/components/QueryOutput/QueryOutput.js:2092]: done

**Review by Shiv-Chata — APPROVED** [review]

**Pull request merged**